OneCommunityGlobal/HighestGoodNetworkApp
# Nahiyan_Added hover effects on tables in dark mode

# Description
Hover effects were added on all tables in dark mode.
Pages: 
- Dashboard (Leaderboard, Team Member Tasks, Badge Report Modal)
- All tables on Reports page
- Team Locations (User Lists)
- User Management (User table, Setup Invitation History Modal)
- Badge Management (Assign badge, Badge Development)
- Projects (Projects table, Members, WBS)
- Teams (Teams table, Members modal)
- Permissions Management (Roles)

## Related PRS (if any):
This frontend PR is related to the dev backend.

## How to test:
1. check into the current branch
2. do `npm install` and `npm run start:local` to run this PR locally
3. Clear site data/cache
4. log in as owner user
5. go to the pages mentioned in the description
6. verify that when hovering over the table row in dark mode, the background color lightens 

## Screenshots or videos of changes:

https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/71025942/8b0c5871-508f-4979-86f7-294572134976
